src/frontend/src/Components/map/RouteSearch.js

```diff
@@ -78,30 +80,36 @@ export default function RouteSearch(props) {
 
   // Rendering
   return (
-    <div className="routing-container">
-      <div className={"typeahead-container typeahead-container--from stacked" + ((!!searchLocationFrom.length || !!searchLocationTo.length) ? ' stacked' : '')}>
-        <span className="location-marker location-marker--from">
-          <FontAwesomeIcon icon={faCircleDot} />
-        </span>
+    <div className={'routing-outer-container'}>
+      {showFilterText && !!searchLocationFrom.length && !!searchLocationTo.length &&
+        <p className={'routing-caption'}>Results below are filtered by this route:</p>
+      }
 
-        <LocationSearch placeholder={'Search starting location'} location={searchLocationFrom} action={updateSearchLocationFrom} />
-      </div>
+      <div className="routing-container">
+        <div className={"typeahead-container typeahead-container--from" + ((!!searchLocationFrom.length || !!searchLocationTo.length) ? ' stacked' : '')}>
```

Comment:
missing "stacked" class broke the styling; please add back to typeahead-container--from.